readthedocs/domaindata/api.py

```diff
@@ -0,0 +1,43 @@
+"""Domain API classes."""
+
+from rest_framework import serializers
+
+from readthedocs.restapi.views.model_views import UserSelectViewSet
+
+from .models import DomainData
+
+
+class DomainDataSerializer(serializers.ModelSerializer):
+    project = serializers.SlugRelatedField(slug_field='slug', read_only=True)
+    version = serializers.SlugRelatedField(slug_field='slug', read_only=True)
+
+    class Meta:
+        model = DomainData
+        fields = (
+            'project',
+            'version',
+            'name',
+            'display_name',
+            'doc_type',
+            'doc_url',
```

Comment:
I suppose this is the _most important_ field we want for the use case you mentioned, as we will want to embed the section when hovering with the mouse or something like that, right?